metron-deployment/roles/kibana/tasks/elasticdump.yml

```diff
@@ -0,0 +1,29 @@
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+---
+- name: Setup Nodesource Yum Repository
+  shell: "curl --silent --location https://rpm.nodesource.com/setup_4.x | bash -"
+
```

Comment:
Sorry, just noticed, this creates a warning ``` [WARNING]: Consider using get_url module rather than running curl``` when running Ansible. Not changing my vote, but if you happen to go back in here while fixing other issues, could you use get_url instead?